### DISCUSSION TITLE
Publish snapshots from CI

### DISCUSSION THREAD

**Discussion by gotson** [xerial/sqlite-jdbc#754] [Ideas]
@xerial i would like to merge #678, but before doing so we need to make sure you added the secrets in the repository:

The following Github secrets will need to be setup in the repository:

SONATYPE_USERNAME: your username for Maven Central
SONATYPE_PASSWORD: your password for Maven Central
GPG_KEY: your GPG key used for signing. It needs to be exported by using gpg --armor --export-secret-keys <keyId>.
GPG_PASSPHRASE: your GPG key passphrase

Can you confirm those have been setup ?

---

In a second step i will look at automatic the releases.

**Comment by xerial** [comment]
ok. I'll add these keys

**Comment by xerial** [comment]
@gotson Added these as repository secrets.

**Reply by gotson** [reply]
> @gotson Added these as repository secrets.

It worked, snapshots are now published automatically 🚀

**Reply by xerial** [reply]
Awesome. Thanks for testing!
